src/Spectre.libHeatmapDataScaling/Normalization.h

```diff
@@ -0,0 +1,45 @@
+/*
+* Normalization.h
+* Class with normalization algorithm implementation.
+*
+Copyright 2018 Daniel Babiak
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+#pragma once
+#include <vector>
+#include "HeatmapDataScalingAlgorithm.h"
+
+namespace Spectre::libHeatmapDataScaling
+{
+    /// <summary>
+    /// Class containing normalization algorithm implementation.
+    /// </summary>
+	class Normalization : public HeatmapDataScalingAlgorithm
+	{
+	public:
+        /// <summary>
+        /// Initializes a new instance of the <see cref="Normalization"/> class.
+        /// </summary>
+        /// <param name="_min">Minimum value of the range.</param>
+        /// <param name="_max">Maximum value of the range.</param>
+		Normalization(const int _min = 0, const int _max = 100);
+
+        /// <summary>
+        /// Method for data normalization to the given range.
+        /// </summary>
+        /// <param name="intensities">Vector of floating point values representing intensities for each point on the heatmap.</param>
+        /// <returns>Vector of normalized floating point values representing intensities for each point on the heatmap.</returns>
+		std::vector<double> scaleData(const gsl::span<double> intensities) override;
+	private:
+		const int min, max;
```

Comment:
`min` and `max` are missing any information about the context. I know this is simple class & functionality, but we all have sometimes bad day, when it's hard to grasp the context on the first look. I think these could be renamed to be more expressive.